Additions as of 4/21/16

mcbratnie15 opened the pull request:
test one additions

mcbratnie15 committed:
Create example.html

mcbratnie15 committed:
Delete example.html

mcbratnie15 committed:
Merge pull request #1 from msu-anthropology/master

Bring tutorial changes

mcbratnie15 committed:
Test 1

Test 1 html document added to files

geyerbri commented:
Before I can merge you changes, you will need to adjust some things. First, your html file should have a name that is all lower-case, uses dashes instead of spaces, and references your content in some way.

Second, make sure none of your images' names contain spaces either. After you rename your images, you will need to make sure the new image name shows up properly in your html file's code.

I will leave this pull request open. When you make the necessary changes described above and synced your application, please comment here to inform me and then we will proceed.

mcbratnie15 committed:
Removed spaces and capital letters

Name Changes removing capitals, spaces and underscores(?)
Removal of one image that was saved as a gif(Sherman logo).

mcbratnie15 commented:
I believe I made all of the changes specified above. But for future reference can I use Underscores instead of dashes for image names?

geyerbri commented:
Yes you can. The more important thing is that you don't use spaces.

I'll commit this set, but I noticed that, after switching the underscores to dashes in one of your images, that image name is now different from the one you use in your html code (on line 78). You'll of course want to eventually fix this before you submit your final version.
